src/FsAutoComplete.Core/ParseAndCheckResults.fs

```diff
@@ -49,6 +55,28 @@ module FCSPatches =
           result.IsSome
       | _ -> false
 
+    member scope.TryRangeOfParenEnclosingOpEqualsGreaterUsage opGreaterEqualPos =
+      let (|InfixAppOfOpEqualsGreater|_|) =
+        function | SynExpr.App(ExprAtomicFlag.NonAtomic, false, SynExpr.App(ExprAtomicFlag.NonAtomic, true, Ident "op_EqualsGreater", actualParamListExpr, _), actualLambdaBodyExpr, _) -> Some (actualParamListExpr, actualLambdaBodyExpr)
```

Comment:
I freaking love active patterns

Reply:
so fun fact: this is the shape of this erroneous usage, so any more cases we want to cover should just be a matter of plugging this into the appropriate `expr` match.

Reply:
Without active patterns, matching the F# AST would be so much harder!